man/repart.d.xml

```diff
@@ -551,6 +551,22 @@
         <xi:include href="version-info.xml" xpointer="v255"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>DefaultSubvolume=</varname></term>
+
+        <listitem><para>Takes an absolute path specifying the default subvolume within the new filesystem.
+        Note that this setting does not create the subvolume itself, that can be configured with
+        <varname>Subvolumes=</varname>.</para>
```

Comment:
why not just imply an appropriate Subvolumes= line if DefaultSubvolume= is not used?

Reply:
Ehh we didn't do that for `Subvolumes=` either (which requires `MakeDirectories=` or `CopyFiles=`), so I just did the same for this to be consistent.